study.md

```diff
@@ -77,7 +83,10 @@ look up Ruby's array and string methods, then check the
     so that your result is `"c-o-f-f-e-e"`.
 
 ```ruby
-# your answer here
+str = "eeffoc"
+str = str.reverse
+str = str.split ""
+str = str.chars.join('-')
```

Comment:
This will cause an error since the previous line turns `str` into an array and there is no `chars` method defined on Array.